liquibase-core/src/main/java/liquibase/change/core/MergeColumnChange.java

```diff
@@ -128,7 +128,7 @@ public SqlStatement[] generateStatements(final Database database) {
         statements.addAll(Arrays.asList(addNewColumnChange.generateStatements(database)));
 
         String updateStatement = "";
-        if (database instanceof MySQLDatabase || database instanceof MariaDBDatabase) {
```

Comment:
Hey @arturobernalg, why are we removing MariaDB check? As far as I know, we can use `CONCAT` function to do so.